lib/cloud_controller/db.rb

```diff
@@ -69,6 +70,15 @@ def self.add_connection_expiration_extension(db, opts)
       db.disconnect
     end
 
+    def self.connection_metrics_extension(db)
+      # only add the metrics for api processes. Otherwise e.g. rake db:migrate would also initialize metric updaters, which need additional config
```

Comment:
**question** Does this also add connection metrics to cc_workers/deployment_updater/clock too right?

Reply:
No it does not. All of the other jobs are rake tasks and are excluded by this line. Adding DB metrics to those other jobs would come with quite some effort. On the worker VMs there is no web server running, which means we cannot emit prometheus metrics there. Currently we are only interested in the DB connection usage of Puma processes. I guess we could add the metrics on other jobs later if needed.